python/python/glide/async_commands/core.py

```diff
@@ -2621,6 +2621,34 @@ async def zmscore(
             await self._execute_command(RequestType.ZMScore, [key] + members),
         )
 
+    async def zdiffstore(self, destination: str, keys: List[str]) -> int:
+        """
+        Calculates the difference between the first sorted set and all the successive sorted sets at `keys` and stores
+        the difference as a sorted set to `destination`, overwriting it if it already exists. Non-existent keys are
+        treated as empty sets.
+
+        When in Cluster mode, all keys must map to the same hash slot.
+
+        See https://valkey.io/commands/zdiffstore for more details.
+
+        Args:
+            destination (str): The key for the resulting sorted set.
+            keys (List[str]): The keys of the sorted sets to compare.
+
+        Returns:
+            int: The number of members in the resulting sorted set stored at `destination`.
+
+        Examples:
+            >>> await client.zdiffstore("my_sorted_set", ["key1", "key2"])
```

Comment:
could be a little more informative,
zadd at the beginning and zrange and the end

Reply:
Sure, added now